### DISCUSSION TITLE
提示登录成功却进不去管理页面

### DISCUSSION THREAD

**Discussion by chouqiao** [AlistGo/alist#2819] [Q&A]
linux服务器刚装的alist，主页点击登录输入账号密码后右上角提示登录成功，却不会跳转管理页面，且页面下方依然显示登录二字。

用serve命令提示有错误
INFO[2022-12-28 22:55:29] reading config file: data/config.json        
INFO[2022-12-28 22:55:29] load config from env with prefix: ALIST_     
INFO[2022-12-28 22:55:29] init logrus...                               
INFO[2022-12-28 22:55:29] Aria2 not ready.                             
INFO[2022-12-28 22:55:29] start server @ 0.0.0.0:5244                  
FATA[2022-12-28 22:55:29] failed to start: listen tcp 0.0.0.0:5244: bind: address already in use 

本地只安装了一个alist，用netstat命令查看端口占用进程pid，ps命令查看pid对应路径显示的就是上面运行命令的路径。

我应该这么解决？

还有用卸载命令后显示成功卸载，但页面还能显示，是有浏览器缓存还有没有删干净？
++++++++++++++++++++++++++++++++
更新，第二天程序可以访问了，任何浏览器访问站点都可以访问管理页面，而且不用输入密码，
用浏览器隐私窗口访问也是如此（不用输入密码，直接管理）
服务器用命令卸载alist，站点依然可以正常访问？   浏览器缓存清理了啊，也换浏览器试了，居然都能正常访问

**Comment by Dashan-IZ** [comment]
我也遇到了一样的问题，不知啊你解决了嘛
。

**Reply by chouqiao** [reply]
> 我也遇到了一样的问题，不知啊你解决了嘛 。

你用的什么面板？宝塔？

**Comment by chouqiao** [comment]
第二天去看程序 它自己好了，但是server命令依然报错FATA[2022-12-29 10:14:23] failed to start: listen tcp 0.0.0.0:5244: bind: address already in use   这如何解决？

**Comment by Dashan-IZ** [comment]
chouqiao ***@***.***>于2022年12月29日 周四10:16写道：

> 第二天去看程序 它自己好了，但是server命令依然报错FATA[2022-12-29 10:14:23] failed to start:
> listen tcp 0.0.0.0:5244: bind: address already in use 这如何解决？
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/2819#discussioncomment-4530028>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUBRJGQX7VJEC3SXYHXBH3WPTYANANCNFSM6AAAAAATLLC6NE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
我解决不了了，好像是缓存的锅
